api/swagger.yaml

```diff
@@ -4082,6 +4082,11 @@ paths:
           description: "If the container is running, kill it before removing it."
           type: "boolean"
           default: false
+        - name: "link"
```

Comment:
I am wondering why https://github.com/docker/docker/blob/master/api/swagger.yaml#L4075 uses `v` instead of `volumes`. While others use `force` instead of `f`.
Seem to mix the shorthand and longhand of option names.